deconstrst/__init__.py

```diff
@@ -20,7 +23,8 @@ def main():
             config.apply_file(cf)
 
     # Lock source and destination to the same paths as the Makefile.
-    srcdir, destdir = '.', '_build/deconst'
+    srcdir = '.'
+    destdir = os.path.join('_build', get_conf_builder())
```

Comment:
Oh, you could probably do:

```python
srcdir = directory
```

Rather than chdiring around. This is fine, though.